vespalib/src/vespa/vespalib/util/threadstackexecutorbase.h

```diff
@@ -35,6 +34,8 @@ class ThreadStackExecutorBase : public SyncableThreadExecutor,
 {
 public:
     using init_fun_t = std::function<int(Runnable&)>;
+    using LockGuard = std::unique_lock<std::mutex>;
```

Comment:
prefer using new types directly in the code to avoid confusion with old concepts

Reply:
Fixed